docs/index.asciidoc

```diff
@@ -0,0 +1,209 @@
+:plugin: dissect
+:type: filter
+
+[id="plugins-{type}-{plugin}"]
+
+=== Dissect
+
+include::plugin_information.asciidoc[]
+include::../../../logstash/docs/include/plugin_header.asciidoc[]
```

Comment:
I'm a little confused here about where everything will live for the doc build. Are we planning to phase out the logstash-docs repo?